lib/api/core/auth/passwordManager.js

```diff
@@ -0,0 +1,30 @@
+/****
+First really simple version of Password Manager
+@TODO :
+- make secret, algo, digest variables configurable
+- improve globally the manager to enable more features
+*****/
+
+var
+  q = require('q'),
+  crypto = require('crypto');
+
+module.exports = function PassportManager () {
+
+  this.secret = 'jbsB8r69PFP39KdLtjVr25Z22';
```

Comment:
Note: Can easily be done by adding this secret in .kuzzlerc. Then, can access with `kuzzle.config.myVariable`